test/functional/core/server_spec.lua

```diff
@@ -191,9 +192,23 @@ describe('server', function()
     if t.is_os('win') then
       return
     end
-    local peer_addr = n.new_pipename()
-    local client =
-      n.new_session(true, { args = { '--clean', '--listen', peer_addr, '--embed' }, merge = false })
+
+    local peer_temp = n.new_pipename()
+
+    -- Find the last instance of '/' to extract the servername
+    local slash_ind = peer_temp:match('.*/()')
+    if slash_ind == nil then
+      slash_ind = 0
+    end
+    local peer_name = peer_temp:sub(slash_ind)
```

Comment:
why not use a capture group in the `match()` ? that would avoid all this index fiddling.